### PR TITLE
Update Wallet API specification draft document

### PR DESCRIPTION
**Pull request by KtorZ**
# Issue Number

None.

# Overview

Latest discussion with frontend'd folks has yield a couple of interesting points.
This draft is the final draft and may change as we start implementing the various
features.

# Comments

Preview of the file right here:

https://rebilly.github.io/ReDoc/?url=https://raw.githubusercontent.com/input-output-hk/cardano-wallet/dd0b38b201bc823f3576555c82f249c111857ab4/specifications/api/swagger.yaml

### CODE REVIEW

File: specifications/api/swagger.yaml [hunk 1]
```diff
@@ -102,15 +107,20 @@ walletMnemonicSentence: &walletMnemonicSentence
     format: bip-0039-mnemonic-word{english}
   example: ["squirrel", "material", "silly", "twice", "direct", "slush", "pistol", "razor", "become", "junk", "kingdom", "flee"]
```
Comment (piotr-iohk) [inline specifications/api/swagger.yaml]: small thing but this example has 12 words not 15 ;)

Comment (KtorZ) [reply specifications/api/swagger.yaml]: Whoops.

**Comment by piotr-iohk** [comment]
Probably it was already discussed and ruled as no, but don't we need the endpoint for getting details of a single wallet `GET \wallets\{wallet_id}`?

**Comment by KtorZ** [comment]
> Probably it was already discussed and ruled as no, but don't we need the endpoint for getting details of a single wallet GET \wallets\{wallet_id}?

Thanks for catching this. This is indeed an oversight.

**Review by piotr-iohk — APPROVED** [review]
lgtm :+1:

**Pull request merged**